features/conference/components/Conference.js

```diff
@@ -18,13 +27,25 @@ class Conference extends Component {
      */
     render() {
         return (
-            <ConferenceContainer>
+            <ConferenceContainer onPress = { this.onPress.bind(this) }>
```

Comment:
We bind function in constructor in order not to rebind on re-render:
```javascript
constructor(props) {
    .....
    this._onPress = this._onPress.bind(this);
}
```
Also such private methods are worth naming with _.